static/app/views/eventsV2/table/quickContext.tsx

```diff
@@ -196,13 +153,113 @@ function IssueContext(props: IssueContextProps) {
   );
 }
 
+type ContextProps = {
+  children: React.ReactNode;
+  contextType: ContextType;
+  dataRow: EventData;
+  organization?: Organization;
+};
+
+export function QuickContextHoverWrapper(props: ContextProps) {
+  const api = useApi();
+  const [ishovering, setisHovering] = useState<boolean>(false);
+  const [error, setError] = useState<boolean>(false);
+  const [loading, setLoading] = useState<boolean>(true);
+  const [data, setData] = useState<Group | null>(null);
+  const delayOpenTimeoutRef = useRef<number | undefined>(undefined);
+
+  const handleHoverState = () => {
+    setisHovering(prevState => !prevState);
+  };
+
+  const fetchData = () => {
+    if (!data) {
+      const params = getRequestParams(
+        props.dataRow,
+        props.contextType,
+        props.organization
+      );
+      api
+        .requestPromise(params.path, params.options)
+        .then(response => {
+          setData(response);
+          if (isIssueContext(props.contextType)) {
+            GroupStore.add([response]);
+          }
+        })
+        .catch(() => {
+          setError(true);
+        })
+        .finally(() => {
+          setLoading(false);
+        });
+    }
+  };
+
+  const handleMouseEnter = () => {
+    handleHoverState();
+    delayOpenTimeoutRef.current = window.setTimeout(() => {
+      fetchData();
+    }, DATA_FETCH_DELAY);
+  };
+
+  const handleMouseLeave = () => {
+    handleHoverState();
+    window.clearTimeout(delayOpenTimeoutRef.current);
+  };
+
+  return (
+    <HoverWrapper>
+      {props.children}
+      <StyledHovercard
+        skipWrapper
+        delay={HOVER_DELAY}
+        body={
+          <QuickContext
+            loading={loading}
+            error={error}
+            data={data}
+            contextType={props.contextType}
+            dataRow={props.dataRow}
+          />
+        }
+      >
+        <StyledIconInfo
+          data-test-id="quick-context-hover-trigger"
+          onMouseEnter={handleMouseEnter}
+          onMouseLeave={handleMouseLeave}
+          ishovering={ishovering ? 1 : 0}
+          onClick={e => e.preventDefault()}
+        />
+      </StyledHovercard>
+    </HoverWrapper>
+  );
+}
+
 const ContextContainer = styled('div')`
   display: flex;
   flex-direction: column;
 `;
 
+const StyledHovercard = styled(Hovercard)`
+  ${Body} {
+    padding: 0;
+  }
+  min-width: 300px;
+`;
+
+const StyledIconInfo = styled(IconInfo)<{ishovering: number}>`
```

Comment:
```suggestion
const StyledIconInfo = styled(IconInfo)<{isHovering: boolean}>`
```

I think we should make this a boolean and we should change `ishovering` -> `isHovering`

Reply:
Hey, this might sound a bit silly. I did have it as a boolean but It came with the error: 
<img width="552" alt="Screen Shot 2022-10-31 at 2 35 57 PM" src="https://user-images.githubusercontent.com/60121741/199083811-8e70c7f0-ee18-4a42-87e3-03e467e031cb.png">

I looked into the error and tried using Transient Props: https://styled-components.com/docs/api#transient-props but it didn't seem to work out.

The icon has a color prop, I guess i'll just do smth this: 
<img width="351" alt="Screen Shot 2022-10-31 at 2 49 51 PM" src="https://user-images.githubusercontent.com/60121741/199086416-2f75789a-5245-49a9-b966-e1a25aec624a.png">